src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -1907,4 +1908,9 @@ private EmailTemplateNames getIntervenerSendOrderContestedTemplate(IntervenerTyp
             }
         }
     }
+
+    public void sendContestedReadyToReviewOrderToJudge(NotificationRequest judgeNotificationRequest) {
```

Comment:
Does this new code have any test coverage?

Reply:
[5c1380b](https://github.com/hmcts/finrem-case-orchestration-service/pull/1943/commits/5c1380bb8ed560dd714e5653d9a3d86169911cd8)